src/scenes/Oracle/components/PredictionTxHistory/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import Table, { TableBody, TableCell, TableHead, TableRow } from 'material-ui/Table';
+import Typography from 'material-ui/Typography';
+import { withStyles } from 'material-ui/styles';
+
+import { getLocalDateTimeString } from '../../../../helpers/utility';
+import styles from './styles';
+
+class PredictionTxHistory extends React.PureComponent {
+  render() {
+    const { classes, transactions, options } = this.props;
+
+    if (!transactions.length || !options.length) {
+      return null;
```

Comment:
do you think we should show the empty table regardless if they have txs or not? thinking it might be weird to have a whole section just pop up after they do a tx?

Reply:
Mmm let's add a note instead of hiding the whole thing then.

Reply:
sounds good :dog: